packages/camera/lib/camera.dart

```diff
@@ -513,4 +513,12 @@ class CameraController extends ValueNotifier<CameraValue> {
       await _eventSubscription?.cancel();
     }
   }
+
+  Future<void> zoomIn() async {
```

Comment:
Would it be better for a user to be able to set the zoom directly. Otherwise a user would have to call `zoomIn()` or `zoomOut` multiple times when initializing the camera.